src/components/custom-field-input-date/custom-field-input-date.jsx

```diff
@@ -36,7 +36,8 @@ function presentDateString(dateString) {
     return '';
   }
 
-  const date = new Date(Date.parse(dateString));
+  const timezone = new Date().toLocaleTimeString('en-us', { timeZoneName: 'short' }).split(' ')[2];
+  const date = new Date(`${dateString}  00:00:00 ${timezone}`);
   return date.toLocaleDateString(undefined, { month: 'short', year: 'numeric', day: 'numeric' });
```

Comment:
@juanca, so we are using `toLocaleDateString` to present the date as a string, Eg. 'May 10, 1992'. However, we made the interface for this component such that the input is a date in the format, "YYYY-MM-DD". The downside to this is there is no timezone information.

`new Date(Date.parse(dateString))` will assume that the hour and minutes are `00:00` and the timezone is UTC. If we call `toLocaleDateString` we could see a different date than the date of the users timezone.

Reply:
So using Javascript's `Date` is really nice here; it would just be a lot nicer if we also used a javascript date object as the prop for value.

Reply:
Do we think we will have the same issue in Calendar?

https://github.com/mavenlink/design-system/blob/82662853c4b4cf99cad5f8356b4690c7410c7ee4/src/components/calendar/calendar.jsx#L67

https://github.com/mavenlink/design-system/blob/82662853c4b4cf99cad5f8356b4690c7410c7ee4/src/components/calendar/calendar.jsx#L74

Also, how are you verifying the changes work? Is there a trick to changing the timezone on your development environment or testing environment?

Reply:
I don't think that has the same issue with the way we pull the date number out of the date object in calendar.

I was kind of lucky and was fixing this bug during the 9 hour-ish window where I think it can occur. But I am sure you could do some clock-trickery to replay the bug.